catalog/core/catalog-core-directorymonitor/src/test/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitorIT.java

```diff
@@ -378,7 +378,8 @@ private Option initContentDirectoryMonitorConfig() {
         new BundleInfo("commons-codec", "commons-codec"),
         new BundleInfo("org.apache.cxf", "cxf-rt-rs-security-jose"),
         new BundleInfo("org.apache.cxf", "cxf-rt-rs-security-jose-jaxrs"),
-        new BundleInfo("org.apache.cxf.services.sts", "cxf-services-sts-core"));
+        new BundleInfo("org.apache.cxf.services.sts", "cxf-services-sts-core"),
+        new BundleInfo("javax.cache", "cache-api"));
```

Comment:
Correct, if the CDM bundle has it embedded it should not need to be wrapped in a bundle info. That said, if there is something else that requires `javax.cache` then this would be necessary, or we embedded it again there.